src/aws-cpp-sdk-core/include/aws/core/config/AWSProfileConfig.h

```diff
@@ -93,6 +93,18 @@ namespace Aws
                 return iter->second;
             }
 
+            inline const Aws::String& GetServicesName() const {
+                auto iter = m_allKeyValPairs.find("services");
+                static const Aws::String empty;
+                return (iter != m_allKeyValPairs.end()) ? iter->second : empty;
```

Comment:
This likely not how we should be returning the idea of "lack of service endpoint". likely [Crt::Optional](https://github.com/awslabs/aws-crt-cpp/blob/main/include/aws/crt/Optional.h) is a good candidate for representing "lacking of value" instead of empty string.